lib/all-badge-examples.js

```diff
@@ -1135,7 +1135,15 @@ const allBadgeExamples = [
         keywords: [
           'PHP'
         ]
-      }
+      },
+      {
+        title: 'PHP version from PHP-Eye',
+        previewUri: '/php-eye/symfony/symfony.svg',
+        keywords: [
+          'PHP',
+          'php-eye'
```

Comment:
Since these keywords are included in the title, there's no need to list them.

Reply:
And why then did we mention them [here](https://github.com/badges/shields/blob/master/lib/all-badge-examples.js#L1125)?

Reply:
Perhaps it was missed. Those could be removed too. Might be good to automate this check in a unit test.